index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<title>Space Ripples Reveal Big Bang's Smoking Gun</title>
+</head>
+<body>
+
+  <nav class="top-nav">
+  			<div class="row top-sub-nav">
+  				<div class="col">
+						<button class="trans-btn d-inline"><i class="fa fa-bars pryt1" aria-hidden="true" style="color:#464646;"></i></button>
+            <button class="trans-btn d-inline"><i class="fa fa-search" aria-hidden="true"></i></button>
+            <p class="cat-title d-inline"><strong>SPACE & COSMOS</strong></p>
+					
+  				</div>
+
+  				<div class="col logo-col">
+  					<img src="image/newyorktimes.png" class="img-fluid logo" alt="new york times logo">
+  				</div>
+
+  				<div class="col">
+
+  					 		<button class="btn btn-blue btn-login" type="submit">Login</button>
+  					 	
+  				</div>
+
+			</div>
+
+		</nav>
+
+ <div class="middle-section">
+  <div class="row">
+  <h1 class="top-title"><em>Space Ripples Reveal <br> Big Bang's Smoking Gun</em></h1>
+  <img src="image/BigBang.jpg" alt="Smoking gun" class="img-fluid">
```

Comment:
`REQUIRED`

All images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video